nomad/periodic.go

```diff
@@ -278,10 +278,10 @@ func (p *PeriodicDispatch) removeLocked(jobID structs.NamespacedID) error {
 // subsequent eval.
 func (p *PeriodicDispatch) ForceRun(namespace, jobID string) (*structs.Evaluation, error) {
```

Comment:
I realize this is an existing method name, but I wonder if calling this `ForceRun` rather than `ForceEval` is misleading.